x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -117,46 +150,116 @@ export function XyToolbar(props: VisualizationToolbarProps<State>) {
+              defaultMessage: 'Fill missing values',
+            })}
           >
-            <EuiFormRow
-              display="columnCompressed"
-              label={i18n.translate('xpack.lens.xyChart.fittingLabel', {
-                defaultMessage: 'Fill missing values',
-              })}
+            <EuiToolTip
+              anchorClassName="eui-displayBlock"
+              content={
+                !hasNonBarSeries &&
+                i18n.translate('xpack.lens.xyChart.fittingDisabledHelpText', {
+                  defaultMessage:
+                    'This setting only applies to line charts and unstacked area charts.',
+                })
+              }
             >
-              <EuiSuperSelect
-                compressed
-                disabled={!hasNonBarSeries}
-                options={fittingFunctionDefinitions.map(({ id, title, description }) => {
-                  return {
-                    value: id,
-                    dropdownDisplay: (
-                      <>
-                        <strong>{title}</strong>
-                        <EuiText size="xs" color="subdued">
-                          <p>{description}</p>
-                        </EuiText>
-                      </>
-                    ),
-                    inputDisplay: title,
-                  };
-                })}
-                valueOfSelected={props.state?.fittingFunction || 'None'}
-                onChange={(value) => props.setState({ ...props.state, fittingFunction: value })}
-                itemLayoutAlign="top"
-                hasDividers
-              />
-            </EuiFormRow>
-          </EuiToolTip>
+              <>
+                <EuiFormRow
+                  display="columnCompressed"
+                  label={i18n.translate('xpack.lens.xyChart.fittingLabel', {
+                    defaultMessage: 'Fill missing values',
+                  })}
+                >
+                  <EuiSuperSelect
+                    compressed
+                    disabled={!hasNonBarSeries}
+                    options={fittingFunctionDefinitions.map(({ id, title, description }) => {
+                      return {
+                        value: id,
+                        dropdownDisplay: (
+                          <>
+                            <strong>{title}</strong>
+                            <EuiText size="xs" color="subdued">
+                              <p>{description}</p>
+                            </EuiText>
+                          </>
+                        ),
+                        inputDisplay: title,
+                      };
+                    })}
+                    valueOfSelected={props.state?.fittingFunction || 'None'}
+                    onChange={(value) => props.setState({ ...props.state, fittingFunction: value })}
+                    itemLayoutAlign="top"
+                    hasDividers
+                  />
+                </EuiFormRow>
+                <EuiHorizontalRule />
```

Comment:
This needs to be outside of the tooltip and form row.
```suggestion
                <EuiHorizontalRule size="s" />
```

Reply:
You mean `margin="s"`?

Reply:
Yep 😆